gluster/fs/pkg/allocator/minmax.go

```diff
@@ -0,0 +1,191 @@
+/*
```

Comment:
Yes, these were copied from aws/efs allocator. I hope these should be in `lib` directory, nice!